### ISSUE TITLE
Delete

### ISSUE THREAD

**Issue by webguyio** [ericcornelissen/tmp#201]

**Comment by mondeja** [comment]
This approach will not help so much because our icons are downloaded through many ways, giving the fact that the package is released in npm, Packagist, CDNs... Also, I'm not sure if you are trying to sort by brand popularity or brand icon popularity, could you clarify that?

I'm not totally against this, but I think it would be out of the scope of the project for now, but you always can create an up to date popularity data file as a [third party extension](https://github.com/simple-icons/simple-icons#third-party-extensions) that we could consume to sort by icon or brand popularity.

**Comment by bhadaway** [comment]
No, I would suggest completely ignoring npm, Packagist, CDNs, or anything else developer-based. That would be a can of worms indeed, and only serve to be counterproductive.

I'm speaking to regular folk, downloading straight from the simpleicons.org website only. Trying to discern between brand and icon popularity (which may be one in the same) would probably be a fool's errand.

Rather, let's throw out the word "popularity" altogether and use instead, "Most Downloaded." Naturally, the obvious ones would rise to the top (Facebook, Twitter, etc.), but again, the real value lies in the top 10, top 100, top 1000, and going even further on this path, to be able to declare the exact number of icons one wishes to download/load into a project would be pretty useful too.

_Concerning my idea about using this as a metric for culling, that probably would be too dangerous. Still, it could be a useful metric for anecdotal reference._
